pyroomacoustics/room.py

```diff
@@ -787,29 +765,22 @@ class Room(object):
     ray_tracing: bool, optional
         If set to True, the ray tracing simulator will be used along with
         image source model.
-    use_rand_ism: bool, optional
-        If set to True, image source positions will have a small random
-        displacement to prevent sweeping echoes
-    max_rand_disp: float, optional;
-        If using randomized image source method, what is the maximum
-        displacement of the image sources?
     """
 
     def __init__(
-        self,
-        walls,
-        fs=8000,
-        t0=0.0,
-        max_order=1,
-        sigma2_awgn=None,
-        sources=None,
-        mics=None,
-        temperature=None,
-        humidity=None,
-        air_absorption=False,
-        ray_tracing=False,
-        use_rand_ism=False,
-        max_rand_disp=0.08,
+            self,
+            walls,
+            fs=8000,
+            t0=0.0,
+            max_order=1,
+            sigma2_awgn=None,
+            sources=None,
+            mics=None,
+            temperature=None,
+            humidity=None,
+            air_absorption=False,
+            ray_tracing=False,
+            min_phase=False,
```

Comment:
Please document new argument in function docstring.

Reply:
Also, is this only used for the DIRPAT responses ? If I understand correctly this can be use for the band-pass filters also in regular use of pyroomacoustics. Is it taken into account in that case too ?